<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rescource.qrc">
    <normaloff>:/icons/icons/system_software_update.png</normaloff>:/icons/icons/system_software_update.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Installed</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Online</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Changes</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTextEdit" name="textEdit">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionDownloadUpdate">
   <property name="text">
    <string>DownloadUpdate</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rescource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionDownloadUpdate</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
